docs/packaging/creating-a-new-package.md

```diff
@@ -166,12 +166,16 @@ git commit
 
 ### Commit message format for new packages
 
+`git commit` on [an initialized repository](prepare-for-packaging.md#initialize-git-hooks) will automatically open your editor with the correct template.
+Note that lines starting with a `#` will be ignored by Git and do not need to be removed.
+
 There should be a summary line (with the package name), a blank line, and then the rest of the commit message.
 
-- There should at the minimum be a Summary and Test Plan.
+- There should at the minimum be a Summary.
```

Comment:
I was going to quibble with "at the minimum" vs "at a minimum", but apparently this usage is common enough. 

LGTM.